dtls.c

```diff
@@ -130,9 +130,10 @@ memarray_t dtlscontext_storage;
  * cookie                 := n bytes
  * cipher_length          := 2 bytes
  * cipher suites (max)    := 2 bytes + max * 2
+ * empty_renegotiation    := 2 bytes
```

Comment:
I think, the `2 bytes` in `2 bytes + max * 2` are already the new `empty_renegotiation := 2 bytes`.

I guess, this is a rudiment of changing the same code a couple of times over a longer period and also change the order of processing those changes.

Reply:
Oh, ok, I will close this PR.